api/src/main/java/org/openmrs/OrderGroup.java

```diff
@@ -0,0 +1,155 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs;
+
+import java.io.Serializable;
+import java.util.LinkedHashSet;
+import java.util.Set;
+
+/**
+ * Transcational Data for the OrderSet {@link org.openmrs.OrderSet}
+ * Contains the OrderSet to be ordered for a particular patient, in a specific encounter
+ */
+public class OrderGroup extends BaseOpenmrsData implements Serializable {
+	
+	public static final long serialVersionUID = 72232L;
+	
+	private Integer orderGroupId;
+	
+	private Patient patient;
+	
+	private Encounter encounter;
+	
+	private Set<Order> orders;
+	
+	private OrderSet orderSet;
+
+	/**
+	 * Gets the orderGroupId
+	 *
+	 * @return the orderGroupId
+	 */
+	public Integer getOrderGroupId() {
+		return orderGroupId;
+	}
+
+	/**
+	 * Sets the orderGroupId
+	 *
+	 * @param orderGroupId the orderGroupId to set
+	 */
+	public void setOrderGroupId(Integer orderGroupId) {
+		this.orderGroupId = orderGroupId;
+	}
+
+	/**
+	 * Gets the patient
+	 *
+	 * @return the patient
+	 */
+	public Patient getPatient() {
+		return patient;
+	}
+
+	/**
+	 * Sets the patient
+	 *
+	 * @param patient the patient to set
+	 */
+	public void setPatient(Patient patient) {
+		this.patient = patient;
+	}
+
+	/**
+	 * Gets the encounter
+	 *
+	 * @return the encounter
+	 */
+	public Encounter getEncounter() {
+		return encounter;
+	}
+
+	/**
+	 * Sets the encounter
+	 *
+	 * @param encounter the encounter to set
+	 */
+	public void setEncounter(Encounter encounter) {
+		this.encounter = encounter;
+	}
+
+	/**
+	 * Gets the orders
+	 *
+	 * @return the orders
+	 */
+	public Set<Order> getOrders() {
+		if (orders == null) {
+			orders = new LinkedHashSet<Order>();
```

Comment:
@wluyima are you suggesting this should be SortedSet instead of Set? Or are you suggesting something else?